producers/mysql-producer/src/main/java/com/flipkart/aesop/runtime/producer/eventhelper/BinLogEventHelper.java

```diff
@@ -0,0 +1,20 @@
+package com.flipkart.aesop.runtime.producer.eventhelper;
+
+import com.google.code.or.common.glossary.Column;
+import com.google.code.or.common.glossary.Row;
+
+import java.util.List;
+
+/**
+ * Created by akshit.agarwal on 11/03/16.
+ */
+public class BinLogEventHelper
+{
+    public static void appendColumnToRow(Row actualRow, Column columnToAppend)
```

Comment:
Sorry my bad. This is not being used. (have removed the usages in last review comment, missed removing the file)